src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -42,7 +42,16 @@ object AudioApiProperties extends LazyLogging {
   lazy val MigrationUser = get("MIGRATION_USER")
   lazy val MigrationPassword = get("MIGRATION_PASSWORD")
 
-  lazy val AudioUrlContextPath = "/audio"
+  lazy val StorageSecretKey = get("STORAGE_SECRET_KEY")
+  lazy val StorageAccessKey = get("STORAGE_ACCESS_KEY")
+  lazy val StorageName = get("STORAGE_NAME")
+
+  lazy val AudioUrlContextPath = "audio"
```

Comment:
Tok lang tid før jeg skjønte hva denne var. Iflg koden benyttes den for å lage en katalog i S3-bøtten hvor audio-filene lagres. Usikker på det er nødvendig i det hele tatt med et nivå i den bøtten? Uansett, så bør denne få et litt bedre navn. ContextPath er et ladet ord og assosieres med en http-url.